src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -44,10 +44,6 @@ internal static class FeatureOnOffOptions
             nameof(FeatureOnOffOptions), nameof(AutoFormattingOnTyping), defaultValue: true,
             storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.Auto Formatting On Typing"));
 
-        public static readonly PerLanguageOption<bool> AutoFormattingOnReturn = new PerLanguageOption<bool>(
```

Comment:
this is editor option visible to users

![image](https://user-images.githubusercontent.com/1333179/54193216-da407d80-4476-11e9-9f11-e02678eef2ab.png)

why are we deleting support for this without any design meeting? @jinujoseph we should discuss deleting support for this option and if we delete this, we should delete the option from option page.